### PR TITLE
feat: implement getIdentity v0

### PR DESCRIPTION
**Pull request by joshuaVayer**

### CODE REVIEW

**Comment by joshuaVayer** [comment]
This PR requires the PR #1 to be merged on main and current branch rebased. Internal dependencies were enabled through PR #1

File: packages/browser/src/authn.ts [hunk 1]
```diff
@@ -47,6 +47,7 @@ export const getSessionCredentials = ({
     environmentId,
     origin
   };
+  console.log("credentials", credentials)
```
Comment (dbrrt) [inline packages/browser/src/authn.ts:50]: console.log statements aren't supposed to stay in the library code in general, I assume you left it for debugging purpose.

Comment (joshuaVayer) [reply packages/browser/src/authn.ts:50]: Exactly, pull request is currently in WIP 😉

**Comment by joshuaVayer** [comment]
@dbrrt based on what was done through this commit: https://github.com/authdog/web-sdk/commit/7cdea5f0347f6c4954fc169c45f1562ec883fb9a should we close this PR

**Comment by dbrrt** [comment]
@joshuaVayer Yes it should be closed

**Pull request closed without merging**